cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
```

Comment:
You could use `ensure_connection()` to make sure that the connection is available.